core/api/serializers/cp_report.py

```diff
@@ -233,7 +233,8 @@ def _create_history(self, cp_report, request_user):
         history_data["country_programme_report_id"] = cp_report.id
         history_data["report_version"] = 1
         history_data["updated_by_id"] = request_user.id
-        history_data["updated_by_username"] = request_user.username
```

Comment:
I think we should keep this info even if we don't show it in the frontend.
Sometimes the request.user may be different from the reporting_officer and I think is important to know who made the request for update

Reply:
We are saving the username in a separate field; actually `updated_by_username` is a read-only field on the serializer. :D